tests/e2e/Services/Account/AccountCustomClientTest.php

```diff
@@ -1314,7 +1315,17 @@ public function testBlockedAccount(): array
             'status' => false,
         ]);
 
+        $this->assertEquals(500, $response['headers']['status-code']);
```

Comment:
Why does this return 500?

Reply:
Shouldn't it be 500 because the user is blocked?

Reply:
No, 500 is a server error. It should return 200 for success